site/_posts/2016-11-28-fixing-bugs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Last Week in ManageIQ: Fixing bugs and fixing bugs"
+author: skateman
+date: 2016-11-28 10:58:27 +01:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello! [Dávid](https://github.com/skateman) here from the UI team with
+some updates about the changes since last week.
+
+## Featured
+
+We are still focused on the stabilization of the upcoming Euwe release.
+In numbers this means 91 commits merged from 88 PRs created by 40 of our
+contributors since last week. If you are interested, check out the list
+of [open issues]((https://github.com/manageiq/manageiq/issues)) to help
+us in increasing these numbers.
+
+## Improved
+
+Chart widgets on the dashboard are sometimes displaying too many information
+on a too tiny place, making some data unreadable. There was a primitive
+zooming feature available, but it was not always helpful. Now thanks to
+our charts expert [Robin](https://github.com/PanSpagetka) the zoomed version
+uses [a full screen modal](https://github.com/ManageIQ/manageiq/pull/12811).
+
+There were also some improvements in the Automate part of our codebase.
+For example [Patrik](https://github.com/pkomanek) continued the [refactoring
+of automate methods](https://github.com/ManageIQ/manageiq/pull/12523). His
+victim was the `available_flavors` method for `miq_ae_orchestration`.
+
+## Fixed
+
+In these though days almost half of the number of PRs have the word "fix"
+in their title. This means it is extremely hard to select which one to
+present here.
+
+We had some issues with infinite spinners on the Reports page when logged
+in as a non-admin user. The problem was with tree unlocking after an AJAX
+change on the page. Thanks to our [Happy Martin](https://github.com/himdel)
+it is now [fixed](https://github.com/ManageIQ/manageiq/pull/12846).
+
+The select box for policy profile selection on the policy simulation screen
+had a fixed size and therefore, it could not properly display longer strings.
+The cleverness of [Milan](https://github.com/mzazrivec) came handy now, as he
```

Comment:
Wondering if s/came handy now/came in handy/ ? Though I am not sure if that is just a Britishism :wink: